framework/src/main/java/org/checkerframework/common/basetype/BaseTypeChecker.java

```diff
@@ -584,6 +587,26 @@ private void printStoredMessages(CompilationUnitTree unit) {
         if (messageStore != null) {
             for (CheckerMessage msg : messageStore) {
                 super.printOrStoreMessage(msg.kind, msg.message, msg.source, unit);
+                this.printStackTrace(msg.trace);
+            }
+        }
+    }
+
+    /**
+     * Prints given stack trace of runtime exception if -AdumpOnErrors flag is passed as compilation
+     * argument
+     *
+     * @param trace stack trace when the checker encounters an error
+     */
+    private void printStackTrace(StackTraceElement[] trace) {
+        boolean dumpOnErrors =
+                (processingEnv != null
```

Comment:
Instead of all this you can simply call `getBooleanOption`? https://github.com/typetools/checker-framework/blob/master/framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java#L1591

Reply:
I have used getBooleanOption instead of the previous logic, however, in this one, I have to provide a boolean value to the command line option for the flag to function.
Ex : javac -AdumpOnErrors=true -processor org.checkerframework.checker.nullness.NullnessChecker -implicit:class Test.java